api/src/test/resources/io/kaoto/backend/api/resource/eip.kamelet.yaml

```diff
@@ -33,11 +33,11 @@ spec:
               completion-mode: MANUAL
               steps:
               - throttle:
-                  caller-runs-when-rejected: false
                   constant: '5'
                   executor-service: myExecutor
-                  reject-execution: false
                   async-delayed: true
+                  caller-runs-when-rejected: false
```

Comment:
are there changes in this file related to this Pull request?
(I think not but given that there are several changes, i'm not sure)

Reply:
Some fields changed the order.

Reply:
relates to https://github.com/KaotoIO/kaoto-backend/pull/510#discussion_r1126491779